### ISSUE TITLE
Xcode SPM unable resolve Firebase dependency

### ISSUE THREAD

**Issue by InfinityCode777** [firebase/firebase-ios-sdk#11523]
### Description

After chose to refresh the packages and I got that error which says "failed downloading 'https://dl.google.com/firebase/ios/swiftpm/10.10.0/FirebaseAnalytics.zip' which is required by binary target 'FirebaseAnalytics': "

I paste link to the browser, the same result, I was unable to download the code too

I started another empty Xcode project and added Firebase via SPM but I got stuck on the downloading screen and has been here for more than 20mins and I got an error finally (see attached snapshots)

### Reproducing the issue

* Start an empty Xcode project 
* Add Firebase using SPM

### Firebase SDK Version

8.9.0, 10.10.0, 10.11.0

### Xcode Version

14.3

### Installation Method

Swift Package Manager

### Firebase Product(s)

Crashlytics

### Targeted Platforms

iOS

### Relevant Log Output

```shell
failed downloading 'https://dl.google.com/firebase/ios/swiftpm/8.9.0/FirebaseAnalytics.zip' which is required by binary target 'FirebaseAnalytics': downloadError("Error Domain=NSURLErrorDomain Code=-1004 \"Could not connect to the server.\" UserInfo={_kCFStreamErrorCodeKey=61, NSUnderlyingError=0x600003ca8ea0 {Error Domain=kCFErrorDomainCFNetwork Code=-1004 \"(null)\" UserInfo={_NSURLErrorNWPathKey=satisfied (Path is satisfied), interface: en0[802.11], ipv4, dns, _kCFStreamErrorCodeKey=61, _kCFStreamErrorDomainKey=1}}, _NSURLErrorFailingURLSessionTaskErrorKey=LocalDownloadTask <CF8F2F8B-1EA1-46C7-A530-FA3F8425CA69>.<1>, _NSURLErrorRelatedURLSessionTaskErrorKey=(\n    \"LocalDownloadTask <CF8F2F8B-1EA1-46C7-A530-FA3F8425CA69>.<1>\"\n), NSLocalizedDescription=Could not connect to the server., NSErrorFailingURLStringKey=https://dl.google.com/firebase/ios/swiftpm/8.9.0/FirebaseAnalytics.zip, NSErrorFailingURLKey=https://dl.google.com/firebase/ios/swiftpm/8.9.0/FirebaseAnalytics.zip, _kCFStreamErrorDomainKey=1}")
```

### If using Swift Package Manager, the project's Package.resolved

Unable to get this far, failed to download the source code
<img width="1088" alt="Screenshot 2023-07-10 at 4 37 02 PM" src="https://github.com/firebase/firebase-ios-sdk/assets/8001169/2f27503e-94d8-4f22-b2a5-610e1d1b4076">
<img width="1094" alt="Screenshot 2023-07-10 at 4 40 59 PM" src="https://github.com/firebase/firebase-ios-sdk/assets/8001169/84d65620-34d4-468f-a7ce-32af32cc1a03">

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by ncooke3** [comment]
The could be a transient network error. Are you able to manually download the resource at https://dl.google.com/firebase/ios/swiftpm/8.9.0/FirebaseAnalytics.zip? If so, I suspect, there is something weird going on with Xcode.

I often reset Xcode state with the following (I usually quit Xcode first): 
```
rm -rf ~/Library/Caches/org.swift.swiftpm/repositories
rm -rf ~/Library/Developer/Xcode/DerivedData
```

**Comment by InfinityCode777** [comment]
Hi, Nik

Thanks for getting back to me. This behavior might not be irrelevant to both Xcode and Google side. I was working in a coffee shop and using their guest wifi. I went back to my apt and tried it again and everything works perfectly. At the moment, I am writing this post in the same coffee shop. I tried it and I am able to download the zip file using the link (I was not even able to do this yesterday). But still I got a failure downloading the packages using SPM in Xcode. I will close this issue for now since, as I said, might not relevant to either Xcode or dl.google.com. 

<img width="1076" alt="image" src="https://github.com/firebase/firebase-ios-sdk/assets/8001169/68e8b2a1-b250-48fb-afe5-f5d61bdb4553">
